packages/front-end/components/Forms/CodeTextArea.tsx

```diff
@@ -205,15 +216,26 @@ export default function CodeTextArea({
   onCtrlEnter,
   wrapperClassName,
   completions,
+  resizable = false,
+  defaultHeight = 200, // for resizable
+  showCopyButton = false,
+  showFullscreenButton = false,
   ...otherProps
 }: Props) {
   // eslint-disable-next-line
   const fieldProps = otherProps as any;
   const { theme } = useAppearanceUITheme();
   const [editor, setEditor] = useState<null | Ace.Editor>(null);
+  const [isFullscreen, setIsFullscreen] = useState(false);
   const containerRef = useRef<HTMLDivElement>(null);
+  const editorUid = useMemo(() => uuid4(), []);
```

Comment:
nit: React does have `useId` but I am not familiar with tradeoffs between uuidv4 and their version of an id.

Reply:
sure, we can use the native hook